template/package.json

```diff
@@ -48,9 +48,7 @@
       "output": "build"
     },
     "files": [
-      "dist/electron",
-      "node_modules/",
-      "package.json"
+      "dist/electron/**/*"
```

Comment:
Wasn't aware of this. I always wondered if there was a way to simplify this. Excellent!